### DISCUSSION TITLE
Leistungsanzeige auf Display nicht runden (1.7 kW -> 17xx W)

### DISCUSSION THREAD

**Discussion by huste511** [tbnobody/OpenDTU#1292] [Ideas]
Wie es der Titel schon sagt, wäre es schöner die Leistungsanzeige auch bei 4 bzw. 5-stelligen Werten nicht zu runden.
Es gehen Informationen verloren und Platz kann nicht das Thema sein.
Anteil der User mit 6-stelligen Werten dürfte überschaubar sein. ;-)

Tagesertrag wird mit Text + 4-stellig + Wh angezeigt.

btw: Bei deutscher Anzeige wird weiterhin ein englischer Punkt als Komma verwendet.

PS: Bei Leistung und Tagesertrag wäre eine Zifferngruppierung mit Punkt als Trennreichen eine feine Sache.

**Comment by brainbarry** [comment]
du meintest wahrscheinlich (1.7 kW -> 17xx **Wh**) ?

**Reply by huste511** [reply]
Nope ... ich meinte W. ;-)
Thx, hab es korrigiert.

**Comment by Tiese** [comment]
Hallo huste511,

du kannst in der Datei: Display_Graphic.cpp die Zeile…
`static const char* const i18n_yield_total_kwh[] = { "total: %.1f kWh", "Ges.: %.1f kWh", "total: %.1f kWh" };`
…z. B. in folgendes ändern…
`static const char* const i18n_yield_total_kwh[] = { "total: %.3f kWh", "Ges.: %.3f kWh", "total: %.3f kWh" };`
Dann ist die Anzeige 3-stellig, nach dem Punkt!
